modules/mixed-arch-modify-machine-set.adoc

```diff
@@ -0,0 +1,69 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-modify-machine-set_{context}"]
+
+= Adding the `arm64` bootimage to your machine set 
```

Comment:
Made a couple fixes based on the discussions from yesterday 😄 

- This module procedure shows how to add the `arm64` bootimage to the machine set, **After** the user has already created their custom machine set.
- I added a link to the `Creating a custom Azure machine set` docs and also added "Generated a custom Azure machine set" as a prerequisite. Which shows the user that they have to create their own machine set before proceeding with this procedure.  
- I left the title because changing it to `Adding a machine set` implies to me that the procedure also shows you how to create a machine set, which is not the case. 

The whole "Configuring a heterogenous cluster" section now should read as follows:
1. Create an azure single architecture azure cluster with the heterogeneous binary. 
2. Create an `arm64` bootimage
3. Create a custom azure machine set.  
4. Modify the machine set to include the `arm` bootimage` (add some kind of deployment step to the procedure)
5. The cluster should now deploy the machine set that uses the `arm64` image

I do think this procedure needs some more details, but I think i'm going to try and set up a meeting with the three of us next week so its easier to discuss. Thank you again for all your help on this 🙂 

Please let me know if something in those steps is wrong.